Website/src/items/content/EdgeWebsite/home/Page Components/Hero.yml

```diff
@@ -1,62 +0,0 @@
-﻿---
-ID: "39743651-6f92-4355-b88b-135cb41353a0"
-Parent: "99536ff1-02bf-5b2e-a94a-1bbba4cf50fa"
-Template: "b33a6d2b-aaa4-567b-8640-18c9e613435e"
```

Comment:
The "Hero" component datasource template does not seem to be used anymore but it was not deleted. Could you please check that is the case and delete it if needed? Thanks.

Reply:
Good one - I have removed the original template that was used as datasource items for the hero component.

Reply:
While that should work and is OK for now, this is not how this page rendering XML value should look like. It should not redefine all placeholders. It should not mention the header, MainNavigation, Footer, SponsorsGrid (and its section) and NewsGrid (and its section) components.